Modules/_testinternalcapi.c

```diff
@@ -2222,7 +2222,7 @@ module_exec(PyObject *module)
     }
 
     if (PyModule_Add(module, "TIER2_THRESHOLD",
-                        PyLong_FromLong(JUMP_BACKWARD_INITIAL_VALUE)) < 0) {
+                        PyLong_FromLong(JUMP_BACKWARD_INITIAL_VALUE + 1)) < 0) {
```

Comment:
I'd rather not have the `+1` here.
Could you add another const `EXCEEDS_TIER2_THRESHOLD = TIER2_THRESHOLD + 1`?
In the tests above, use `EXCEEDS_TIER2_THRESHOLD` where we want to exceed the threshold, and in the places where we need the actual threshold, and you currently have `TIER2_THRESHOLD - 1`,  use `TIER2_THRESHOLD`

In theory, the tests should continue to work if `EXCEEDS_TIER2_THRESHOLD = TIER2_THRESHOLD + 5`.